pkgs/development/tools/build-managers/bazel/bazel_3/default.nix

```diff
@@ -62,6 +64,7 @@ let
       srcs.rules_cc
       srcs.rules_java
       srcs.rules_proto
+      srcs.com_google_protobuf
```

Comment:
We could use ``with srcs;`` here.

Reply:
`with srcs;` is not a net improvement IMO so I'll let it be.

Reply:
If we you add it in line 45 we can remove all those srcs.